app/src/main/res/values-fr/strings.xml

```diff
@@ -385,8 +385,9 @@
     </string-array>
 
     <string-array name="faq_ans">
-        <item>Pour demander un compte de crédit, cliquez sur \ "Signaler la demande de prêt \" sur l\'écran d\'accueil.</item>
-        <item>Vous pouvez afficher les informations de votre profil en cliquant sur l’image de profil sur la page principale de l’application.</item>
+
+        <item>Pour demander un compte de crédit, cliquez sur \ "Signaler la demande de prêt \" sur l'écran d'accueil.</item>
```

Comment:
\ " - remove space in between.
l'écran d'accueil - add escape character at both the apostrophe.